traffic_portal/app/src/common/modules/table/serverCapabilities/table.serverCapabilities.tpl.html

```diff
@@ -34,11 +34,13 @@
             <thead>
             <tr>
                 <th>Name</th>
+                <th>Description</th>
             </tr>
             </thead>
             <tbody>
             <tr ng-click="editServerCapability(sc.name)" ng-repeat="sc in ::serverCapabilities" context-menu="contextMenuItems">
                 <td name="name" data-search="^{{::sc.name}}$">{{::sc.name}}</td>
+                <td name="name" data-search="^{{::sc.description}}$">{{::sc.description}}</td>
```

Comment:
this is a small enough change that you probably don't need to bother with it, but for what it's worth we should convert that table to use the common grid controller at some point.

Reply:
Above comment is a refactor and better off in a new PR. Lets create an issue.